fmt.sh

```diff
@@ -133,6 +136,7 @@ run dade-copyright-headers "$dade_copyright_arg" .
 if [ "$diff_mode" = "true" ]; then
```

Comment:
Whether we are diffing or not does not imply whether we are testing or not.

I suggest setting `prettier_args` to `('--write')` or `('--check')` appropriately above, and using it below.